docs/utilities/feature_flags.md

```diff
@@ -548,6 +548,7 @@ Parameter | Default | Description
 **max_age** | `5` | Number of seconds to cache feature flags configuration fetched from AWS AppConfig
 **sdk_config** | `None` | [Botocore Config object](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html){target="_blank"}
 **jmespath_options** | `None` | For advanced use cases when you want to bring your own [JMESPath functions](https://github.com/jmespath/jmespath.py#custom-functions){target="_blank"}
+**logger** | `None` | Logger to use.  If None is supplied, then a logger will be created.
```

Comment:
maybe add that it needs to have a debug/error/info functions? what do you think?

Reply:
Or we could say it must be compatible with the Python standard **logger** class.

Ideas on wording???

Reply:
logger class must be compatible with the Python standard logger class ?

Reply:
Powertools Logger is also a `logging.Logger` when it comes to statements. Updated the docs to reflect the correct default, and added a note that customers can pass Powertools Logger too.

```markdown
**logger** | `logging.Logger` | Logger to use for debug.  You can optionally supply an instance of Powertools Logger.
```